side-navigation.yaml

```
@@ -117,6 +117,8 @@
       url: /docs/patterns/hero
     - title: Tiered list
       url: /docs/patterns/tiered-list
+    - title: Quote Wrapper
```

If "Tiered list" I guess it should be "Quote wrapper" then
```suggestion
    - title: Quote wrapper
```